### ISSUE TITLE
Implementation Discussion

### ISSUE THREAD

**Issue by BonfireScratch** [padscape/api#1]
@Dandigit I was planning to use MongoDB and Node.js, but I just found out that MongoDB cannot be installed currently (I use Raspbian 4), but I am not entirely sure. Can you help me with that?

##### Note: Is PHPMyAdmin a good option in case I will be unable to install MongoDB?

**Comment by Dandigit** [comment]
@BonfireScratch Which Raspberry Pi model are you using?

**Comment by BonfireScratch** [comment]
@Dandigit Raspberry Pi 3

**Comment by Dandigit** [comment]
@BonfireScratch Looks like you'll need to install Ubuntu on your Pi.
Burn [this image](http://cdimage.ubuntu.com/ubuntu/releases/bionic/release/ubuntu-18.04.2-preinstalled-server-arm64+raspi3.img.xz) to your SD card and set it up with your Pi. Then you'll be able to use the official MongoDB Ubuntu/ARM64 release.

**Comment by Dandigit** [comment]
@BonfireScratch Some links to help with this:
- https://wiki.ubuntu.com/ARM/RaspberryPi
- https://andyfelong.com/2019/03/mongodb-4-0-6-64-bit-on-raspberry-pi-3/

**Comment by BonfireScratch** [comment]
@Dandigit I might consider doing so

**Comment by BonfireScratch** [comment]
@Dandigit I think I will do that, but it will take some time as I am working on other stuff as well

**Comment by BonfireScratch** [comment]
@Dandigit I am commenting from Ubuntu Mate on a Raspberry Pi 3B+!

**Comment by BonfireScratch** [comment]
@Dandigit the Raspberry Pi seems to have zoomed the screen in so much that I can do almost nothing with it... :angry:

**Comment by BonfireScratch** [comment]
@Dandigit @SimpleBinary do you know how can I use port-forwarding?

**Comment by SimpleBinary** [comment]
@BonfireScratch Could you explain what you mean by "do you know how can I use"? It is quite hard to  work out what you are asking here.

**Comment by BonfireScratch** [comment]
@SimpleBinary how can I port forward my REST API application so that the GitHub server can use it?

**Comment by BonfireScratch** [comment]
@Dandigit can you point mi n the right direction about a very strange issue I have?

**Comment by SimpleBinary** [comment]
@BonfireScratch What is the issue?

**Comment by BonfireScratch** [comment]
@SimpleBinary the issue can only be reproduced locally, as I have not made the configurations on the Apache server to handle HTTPS. (I will add images soon) I have added a feature on the front end that, given a URL (eg `padscape.github.io#3`), the app connects to the server and retrieves the content for a 3rd entry given to the database, that contains sample code. While I was writing example code snippets, I noticed that, when reloading the page, some parts were missing. Doing some testing, I believe the issue is in the server side. I have figured out that it occurs due to the presence of single and double quotes, but I cannot figure out why... I first thought it would be because of escape characters, but new screenshots can show that it is not because of that, at least, only.
